opensrp-immunization/src/main/java/org/smartregister/immunization/util/VaccinatorUtils.java

```diff
@@ -871,13 +871,12 @@ public static String getFileName(String fileName, String prefix) {
 
 
     public static List<org.smartregister.immunization.domain.jsonmapping.Vaccine> getVaccineFromVaccineConfigFile(@Nullable android.content.Context context, String vaccines_file) {
-        Map<String, Object> jsonMap = new HashMap<>();
         Class<List<org.smartregister.immunization.domain.jsonmapping.Vaccine>> classType = (Class) List.class;
         Type listType = new TypeToken<List<org.smartregister.immunization.domain.jsonmapping.Vaccine>>() {
         }.getType();
 
         if (specialVaccines == null) {
-            specialVaccines = ImmunizationLibrary.assetJsonToJava(jsonMap, context, vaccines_file, classType, listType);
+            specialVaccines = ImmunizationLibrary.assetJsonToJava(ImmunizationLibrary.getInstance().getVaccinesConfigJsonMap(), context, vaccines_file, classType, listType);
```

Comment:
Also need to confirm the effect of inflating the file into an already existing vaccine config map. Does it delete the existing and replace with the new values or does it append? This is because this method will be invoked multiple times on the same data structure which is static and used globally to cache.

Reply:
It just seems to check if the file contents are already cached. Then the cached values are used, otherwise, we add the filecontents into the map as cache for later use